src/functions/definitions/connector_function_test.ts

```diff
@@ -0,0 +1,80 @@
+import { assertEquals, assertStrictEquals } from "../../dev_deps.ts";
```

Comment:
A lot of the tests in this file validate `FunctionDefinition` logic, rather than `ConnectorFunctionDefinition`

Reply:
The abstract base class was no longer required I've removed it and this test file 💯